fight.h

```diff
@@ -0,0 +1,18 @@
+#include "Fighter.h"
+class Fight {
+public:
+    Fight();
+    Fight(int amount);
+    ~Fight();
+    void addFighterToArray(Fighter fighter,unsigned int INDEX);
+    Fighter getFighterFromArray(int INDEX);
+    void attackOfFighters(Fighter fighter1,Fighter fighter2);
+
+
+
+private:
+    int lenghtOfArray;
+    Fighter* arrayoffighters;
```

Comment:
Harsh variable name. Please, rename from "**arrayoffighters**" to "**arrayOfFighters**".